motr/setup.c

```diff
@@ -2293,6 +2294,11 @@ static int _args_parse(struct m0_motr *cctx, int argc, char **argv)
 				{
 					rctx->rc_fis_enabled = true;
 				})),
+			M0_NUMBERARG('r', "ADDB Record storage size",
+				LAMBDA(void, (int64_t size)
+				{
+					rctx->rc_addb_record_file_size = size;
```

Comment:
Do we need to validate the size?
For example, it should be more than 10MB?
It should be less than 10GB?

Anyway, a reasonable size should be verified.

Reply:
Corrected 47008ff